homeassistant/components/tuya/light.py

```diff
@@ -413,6 +413,7 @@ class TuyaLightEntity(TuyaEntity, LightEntity):
     _color_data_type: ColorTypeData | None = None
     _color_mode: DPCode | None = None
     _color_temp: IntegerTypeData | None = None
+    _attr_name = None
```

Comment:
This is not correct, this will override the description from the entity descriptions.

Reply:
The original bug report mentions that the complaining entity was the TuyaLightEntity, should all other entity descriptions have `name=None,`?

Reply:
Yes

Reply:
Otherwise entity descriptions will be ignored

```suggestion
```